src/content/docs/merchant/multistore.mdx

```diff
@@ -0,0 +1,182 @@
+
+Each store view has its own set of configuration files. The `configs.xlsx` file defines the API endpoints, headers, and other environment-specific data for your production environment. The `configs-stage.xlsx` and `configs-dev.xlsx` files are used for staging and development environments, respectively.
+
+### Placeholders
+
+The `placeholders.xlsx` file stores reusable tokens for text and UI components. This file is used to define the text that is displayed on your storefront. The `placeholders.json` file is generated from this file and is used to replace the tokens in your content files. You can optionally create a `placeholders-overrides.json` file that define text that supersedes the values defined in the `en` store view.
+
+## Step-by-Step
+
+This example adds a new English Canadian store view (`en_ca`) that is similar to the default English store (`en`).
+
+<Tasks>
+
+<Task>
+### Add a new store
+
+Create an `en_ca` store view in the Commerce Admin and perform the following steps in your project.
+
+1. Create the `en_ca` folder under the project root in Sharepoint or Google Docs.
+
+1. Copy the configuration files, placeholders file, and document pages from the `en` directory to the `en_ca` directory.
+
+</Task>
+
+<Task>
+### Update Content Files
+
+Next, localize or translate the files in your project.
+
+1. Translate or localize content files in the `/en_ca/` folders.
+
+1. Ensure all labels in `placeholders.xlsx` are accurately translated into the desired language.
+
+</Task>
+
+<Task>
+### Update environment configurations
+
+Modify the three configuration files to include appropriate Commerce API endpoints and headers, such as the store view.
+
+</Task>
+
+<Task>
+### Set metadata to the chosen market's path
+
+Create a metadata file and configure settings for the new store view.
+
+1. Create a `metadata.xlsx` file in the root of your project.
+
+1. Set the **root** values to the directory where `configs.xlsx` files are located. The default is `/`.
+
+1. Set the **placeholders** to the `placeholders.json` file to use. The default is `/placeholders.json`.
+
+1. (Optional) Create a `placeholders_overrides.json` file if you have overrides that apply to the Canadian store.
+
+   | Key | Value |
+   |---|---|
+   | `Cart.Cart.heading` | `Shopping Cart ({count})` |
+   | `Cart.Cart.editCart` | `Edit` |
+
+</Task>
+
+<Task>
+### Map folders
+
+Update the `fstab.yaml` file of the project to  add folder mapping to the PDP page.
+
+:::note[Code Change Required]
+This is the only code change required and must be merged into the `main` branch to take effect.
+:::
+
+</Task>
+
+<Task>
+### Preview and validate
+
+Finally, preview and publish the new store view.
+
+1. Preview the updated folder `/en_ca/` with [AEM Sidekick]().
+
+1. Validate the storefront experience by accessing the new language/region and testing the rendering, navigation, and data accuracy.
+
+1. Publish the updated folder with the AEM Sidekick folder when ready.
+
```

Comment:
Minor comment: perhaps final step could be to update store-swticher.docx and include link in recommended bulleted structure with title. @anthoula